src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java

```diff
@@ -291,20 +284,22 @@ private String generateTimePartitionCondition(StandardTableDefinition tableDefin
     }
 
     if (partitionFromDate != null) {
+      String columnFromTimestampName = columnName;
       if (LegacySQLTypeName.DATE.equals(columnType)) {
-        columnName = "TIMESTAMP(\"" + columnName + "\")";
+        columnFromTimestampName = "TIMESTAMP(`" + columnFromTimestampName + "`)";
```

Comment:
`columnFromTimestampName` is a confusing name. Can you think of a better/simpler to understand column name?